src/assets/dresses/asian_dress/asian_dress.asset.ts

```diff
@@ -36,6 +62,20 @@ DefineAsset({
 				editedBy: 'ClaudiaMia',
 				license: 'Pandora-Use-Only-v1-or-later',
 			},
+			{
+				part: 'Dragon-Pattern',
+				copyrightHolder: 'Image by rawpixel.com',
+				source: 'https://www.rawpixel.com/services/licenses',
```

Comment:
The source should link to the page of the actual image, not to a "licenses" site.